commands/base.py

```diff
@@ -121,7 +134,7 @@ def check_permissions(self):
 
     def check_channels(self):
         cr = self.channels_required
-        if cr and len(cr) > 0:
+        if cr is not None and len(cr) > 0:
```

Comment:
channels_required is by default empty, this would mean we would have to include EVERY channel for each command which I find tedious and not practical.

Reply:
Nevermind, I just thought about it and what you said makes sense